src/libs/ReportUtils.js

```diff
@@ -1089,32 +1089,46 @@ function getPersonalDetailsForAccountID(accountID) {
     );
 }
 
+function hasOnlyAvatarField(obj) {
+    const keys = _.keys(obj);
+    return keys.length === 1 && keys[0] === 'avatar';
```

Comment:
not sure why. This is safe and very readable. Lodash would be something like this
```
_.isEqual(_.keys(obj), ['avatar']);
```

Reply:
What happens to this personalDetails object post-invite once the API responds. Does it get cleared out? If so, perhaps we can add an explicit `isOptimistc` key to it instead of having to infer it.

Reply:
Not sure what you mean by `cleared out`? But when we search for a non-existing account we see this.
<img width="878" alt="image" src="https://github.com/Expensify/App/assets/3069065/06e3f13b-2572-4dde-af04-db6c25d42cac">

Once we receive the object again post-invite, it includes all the keys. For example,

```
{
    "accountID": 15820244,
    "avatar": "https://d2k5nsl2zxldvw.cloudfront.net/images/avatars/default-avatar_4.png",
    "displayName": "fwefwewfwe@wefnkewn.com",
    "firstName": "",
    "lastName": "",
    "status": null
}

```

Reply:
Sorry, I think I'm confused by what an "invite" account is. 

From what I can see, when I invite a new user, we create an optimistic personalDetails object for that user that looks like this: 
<img width="633" alt="Screenshot 2023-10-13 at 10 35 27 AM" src="https://github.com/Expensify/App/assets/1371996/33c23b9a-0f13-470d-b451-2eddad61517e">

Then when the API responds, the optimistic personalDetail gets removed and the real personalDetail with the real accountID for that user is added: 
<img width="594" alt="image" src="https://github.com/Expensify/App/assets/1371996/c7a95278-2d51-4d43-9cba-ef79cd41a071">

So I'm just trying to understand what this "invite" account that has only the avatar field is and when it would exist. Sorry if I'm missing something obvious!